src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Drivetrain extends SubsystemBase {
+
+  private CANSparkMax leftMotor = new CANSparkMax(0, MotorType.kBrushless);
+  private CANSparkMax rightMotor = new CANSparkMax(1, MotorType.kBrushless);
+
+
+  public Drivetrain() {
+    rightMotor.setInverted(true);
+  }
+
+  public void toggleBrakingMode() {
+    CANSparkMax.IdleMode modeLeft = leftMotor.getIdleMode();
+    CANSparkMax.IdleMode modeRight = rightMotor.getIdleMode();
+
+    if(modeLeft == CANSparkMax.IdleMode.kBrake) {
+      leftMotor.setIdleMode(CANSparkMax.IdleMode.kCoast);
+      rightMotor.setIdleMode(CANSparkMax.IdleMode.kCoast);
+    } else {
+      leftMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
+      rightMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
+    }
+  }
+
+  public void stopDrive() {
+    leftMotor.set(0);
+    rightMotor.set(0);
+  }
+
+  public void arcadeDrive(double xSpeed, double zRotation) {
```

Comment:
This method should be implemented via a call to the `arcadeDrive` method of a `DifferentialDrive`